source/features/conversation-activity-filter.tsx

```diff
@@ -147,17 +151,20 @@ function addWidget(position: Element): void {
 	);
 }
 
-function init(): void {
-	const observer = observe(':is(.gh-header-sticky .meta, .gh-header-meta .flex-auto):not(.rgh-conversation-activity-filter)', {
-		add: addWidget
-	});
-	deinit.push(observer.abort);
+async function init(): Promise<void> {
+	// Work around for `clean-conversation-headers`
+	const mainHeaderSelector = pageDetect.isPR() ? 'relative-time' : 'span[data-issue-and-pr-hovercards-enabled]';
+	await addWidget(`#partial-discussion-header .gh-header-meta :is(clipboard-copy, ${mainHeaderSelector})`);
+	await addWidget('#partial-discussion-header .gh-header-sticky :is(clipboard-copy, relative-time)');
```

Comment:
Since we’re always appending it, can we just use elementReady on `.gh-header-*` elements instead? Maybe that won’t require us to select a specific icon inside.

Reply:
The issue is not being ready. Its where at append the icon.

Unless I misunderstood you.

Reply:
@fregante?